internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -91,7 +87,11 @@ func (v *vmConsoleProxy) WatchClusterTypes() []operands.WatchType {
 }
 
 func (v *vmConsoleProxy) Reconcile(request *common.Request) ([]common.ReconcileResult, error) {
-	if !isEnabled(request) {
+	if request.Instance.Spec.FeatureGates == nil {
```

Comment:
When `.Spec.FeatureGates` is `nil`, the behavior should be exactly the same as if feature gate `DeployVmConsoleProxy` is false.